### PR TITLE
Add formatting expressions for IN model.

### PR DESCRIPTION
**Pull request by norgevz**

### CODE REVIEW

File: model/countries/IN/IN-formatting-rules.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+formatting-rules:
+  address:
+  - street-address-alternative-1
+  - separator: "\n"
+  - postal-code
```
Comment (battre) [inline model/countries/IN/IN-formatting-rules.yaml:5]: This does not correspond to libaddressinput nor my meeting notes.

https://www.upu.int/UPU/media/upu/PostalEntitiesFiles/addressingUnit/indEn.pdf
https://en.wikipedia.org/wiki/Address

There seem to be multiple address formats but none of them put the postal code before the localit1

File: model/countries/IN/IN-formatting-rules.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - separator: " "
+  - locality1
+  - separator: "\n"
+  - admin-area1
+  - separator: "\n"
+  - country-name
+  - skip: country  # redundant with country-name
+  - skip: street-address  # redundant with street-address-alternative-1
+  - skip: admin-area1
```
Comment (battre) [inline model/countries/IN/IN-formatting-rules.yaml:14]: Why do you list "skip: admin-area1" here and include it in line 9?

File: model/countries/IN/IN-formatting-rules.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+  street-location-and-locality2:
```
Comment (battre) [inline model/countries/IN/IN-formatting-rules.yaml:16]: Have you double checked the order here?

File: model/countries/IN/IN-formatting-rules.yaml (hunk 1 above)
Comment (norgevz) [reply model/countries/IN/IN-formatting-rules.yaml]: Sorry I didn't realize I had sent this for review. The PR is ready now. 

I've followed the format presented in the Aadhaar form [1]. It also matches most of the address formats in the links you shared above.

[1] https://5.imimg.com/data5/SELLER/Doc/2021/4/JZ/CF/JJ/126263748/aadhaar-card.pdf

File: model/countries/IN/IN-formatting-rules.yaml (hunk 2 above)
Comment (norgevz) [reply model/countries/IN/IN-formatting-rules.yaml]: Was just WIP. PTAL now.

File: model/countries/IN/IN-formatting-rules.yaml (hunk 3 above)
Comment (norgevz) [reply model/countries/IN/IN-formatting-rules.yaml:21]: I think I have.

File: model/countries/IN/IN-formatting-rules.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  address:
+  - street-location
+  - separator: "\n"
+  - landmark
+  - separator: "\n"
+  - locality2
+  - separator: "\n"
+  - locality1
+  - separator: "\n"
+  - locality1
```
Comment (battre) [inline model/countries/IN/IN-formatting-rules.yaml:11]: 2x locality1?

Comment (norgevz) [reply model/countries/IN/IN-formatting-rules.yaml:9]: Resolved.

**Review by battre — APPROVED** [review]

**Pull request merged**